mpcs/src/basefold/structure.rs

```diff
@@ -242,30 +274,11 @@ where
 {
     pub(crate) sumcheck_messages: Vec<Vec<E>>,
     pub(crate) commits: Vec<Digest<E>>,
-    pub(crate) final_message: Vec<E>,
+    pub(crate) final_message: Vec<Vec<E>>,
     pub(crate) query_opening_proof: QueryOpeningProofs<E>,
     pub(crate) sumcheck_proof: Option<Vec<IOPProverMessage<E>>>,
```

Comment:
It seems `sumcheck_proof` is identical to `sumcheck_messages`, shall we remove `sumcheck_messages` field and keep `sumcheck_proof` only?

Reply:
Yes, add new commit https://github.com/scroll-tech/ceno/pull/894/commits/1e403e80544a644cee56d9d73340c053dcd01cc1 to clean up old `sumcheck_messages` and only keep `sumcheck_proof `